magento/pwa-studio
# [feature]: Basic desktop improvements for Venia - Mega Menu

## Description

Added Mega Menu for Venia.

## Related Issue





Closes #2805.

### Verification Steps


1. Go to the  homepage or any other page
2. Verify Megamenu

## Screenshots / Screen Captures (if appropriate)
### Mega menu - base view
![mega menu - base](https://user-images.githubusercontent.com/11998249/104089785-ef081b00-5271-11eb-9c2f-cfd8116d776d.png)

### Mega menu - active state
![mega menu - active state](https://user-images.githubusercontent.com/11998249/104089798-1959d880-5272-11eb-9561-0a1e3db33474.png)
### Header &Mega menu - offline indicator
![mega menu - offline indicator](https://user-images.githubusercontent.com/11998249/104089818-31c9f300-5272-11eb-9b4d-9aea4e24f56c.png)
### Header & Mega menu - page loading indicator
![mega menu - pge loading indicator](https://user-images.githubusercontent.com/11998249/104089823-3c848800-5272-11eb-868f-484039c3e7a2.png)
### Mega menu - submenu
![mega menu - submenu](https://user-images.githubusercontent.com/11998249/104089828-473f1d00-5272-11eb-900f-b7e11d9a532b.png)
### Mega menu - submenu - only two levels of categories
![mega menu - submenu - only one level](https://user-images.githubusercontent.com/11998249/104089836-5625cf80-5272-11eb-945f-7e18feda68dd.png)
### Mega menu - submenu on small screens
![mega menu - submenu - small screens](https://user-images.githubusercontent.com/11998249/104089845-6342be80-5272-11eb-858a-4b2b98e0ebe7.png)



## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
